apps/thunderhub/data/thubConfig.yaml

```
@@ -0,0 +1,14 @@
+masterPassword: 'moneyprintergobrrr'
+accounts:
+  - name: 'Umbrel'
+    serverUrl: '10.0.0.9:10009'
```

Hey! I think you should use the env variable LND_IP et LND_PORT instead of hard coding the host:port. You can for example set a container env variable for the server url:
```
environment:
    - SERVER_URL: $LND_IP:$LND_PORT
```
```suggestion
    serverUrl: '${SERVER_URL}'
```
Or just set the same env var and use
```suggestion
    serverUrl: '${LND_IP}:${LND_PORT}'
```

---

Hey! Yeah plan on adding the functionality to read env vars inside the config file in a next release but it's currently not possible so had to hard code them